MTM-Backend/src/routes/v1/user/user.router.ts

```diff
@@ -25,7 +25,27 @@ userRouter.get(
 );
 
 /**
- * get a user by id
+ * get a user by email
+ */
+userRouter.get(
+  "/v1/getUserByEmail",
+  async (req: Request, res: Response, next: NextFunction) => {
+    const email = req.body.email;
```

Comment:
We usually don't use req.body for GET method as a convention. It's a pattern to include email as part of the url param. Please see API doc for the specific of the request